src/main/java/edu/sjsu/articlevisualisationbackend/controller/FileUploadController.java

```
@@ -40,6 +43,11 @@ public ResponseEntity<String> uploadPdf(@RequestParam("file") MultipartFile file
 
             String fileText = this.pdfToText.pdf_to_text(filePath);
 
+            int countToken = this.checkPdfSize.check_pdf_size(fileText);
+
+            if (countToken >= 3500)
```

A few suggestions:
- This if statement is better to put in the Service layer. The control layer primarily focuses on handling HTTP requests and responses. It's better to throw an error in the service layer and catch the error in controller layer.
- You don't want to hardcode 3500. You can make a 'final' constant for 3500.
```java
final int WORD_LIMIT = 3500;
```